_assets/js/templates/textBestBetsTemplate.js

```diff
@@ -0,0 +1,13 @@
+const textBestBetsTemplate = (content) => {
+   return `
+    <li class="padding-bottom-5 margin-top-4 usa-prose border-bottom-05 border-base-lightest">
+      <b class="title"><a href="${content.url}">${content.title
+    .replace(/\uE000/g, '<span class="bg-yellow">')
```

Comment:
Could we make a function out of it that says?

Reply:
I can make another template. The reason I did not is that this felt like not a lot of HTML to leave in there, but I'll componentize this as well.

Reply:
Done as well. See new commit.